.github/dependabot.yml

```diff
@@ -45,10 +49,14 @@ updates:
       applies-to: version-updates
       patterns:
       - k8s.io/*
+  commit-message:
+    prefix: "build(deps)"
 - package-ecosystem: github-actions
   directory: /
   schedule:
     interval: daily
   reviewers:
   - package-operator/core-maintainers/dependency-maintainers
   open-pull-requests-limit: 100
+  commit-message:
+    prefix: "build(deps)"
```

Comment:
Only a small change needed from my side, please put a newline at the end of the file :)